### DISCUSSION TITLE
Is it possible to run edge functions as just normal functions that are closer to the database than the user?

### DISCUSSION THREAD

**Discussion by galacticgibbon** [supabase/supabase#13880] [Questions]
For 90% of my edge function tasks, I am calling the database multiple times. So it doesn't make sense for the functions to be hosted closer to the user.

Is it possible to configure edge functions so that they actually run close to the database instead?

**Comment by GaryAustin1** [comment]
You currently can't control the location of where the edge function runs.

**Comment by steve-chavez** [comment]
Database functions might be better suited here https://supabase.com/docs/reference/javascript/rpc
